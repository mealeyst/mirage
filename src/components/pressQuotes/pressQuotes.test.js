import React from 'react'
import { css } from 'styled-components'
import 'jest-styled-components'

import { PressQuotes } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) PressQuotes', () => {
  const createPressQuotes = (props) => {
    return shallowWithTheme(<PressQuotes {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createPressQuotes())
    .toMatchSnapshot()
  })
})
