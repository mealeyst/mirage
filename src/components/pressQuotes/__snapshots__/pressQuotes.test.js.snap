// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) PressQuotes matching the snapshot 1`] = `
<Styled(BasePressQuotes)
  constrained={true}
  header="BUZZZZZZZZZ"
  headerLabel="What's the buzz?"
  padding={true}
  quotes={
    Array [
      Object {
        "id": "new_york_times",
        "name": "The New York Times",
        "quote": "“Materials and details are thoughtful, and the clothes are wildly cute.”",
      },
      Object {
        "id": "today_show",
        "name": "Today Show",
        "quote": "“The clothing is so stylish and so affordable...and the softest quality.”",
      },
      Object {
        "id": "people_magazine",
        "name": "People Magazine",
        "quote": "“They'll be at the top of their playground game with these clothes.”",
      },
      Object {
        "id": "tech_crunch",
        "name": "Tech Crunch",
        "quote": "“Rockets of Awesome is a total no brainer.”",
      },
      Object {
        "id": "fast_company",
        "name": "Fast Company",
        "quote": "“Shopping for kids’ clothes can be fun.”",
      },
      Object {
        "id": "parents_magazine",
        "name": "Parent's Magazine",
        "quote": "“You can’t buy these clothes in stores.”",
      },
      Object {
        "id": "la_times",
        "name": "The L.A. Times",
        "quote": "“Obsessively kid friendly.”",
      },
      Object {
        "id": "new_york_post",
        "name": "The New York Post",
        "quote": "“The witty graphics, throwback bombers and metallic-accented tutus from this fashion-forward label are all about encouraging kids 'to be their authentic selves.'",
      },
    ]
  }
  theme={
    Object {
      "base": undefined,
      "breakpoints": Object {
        "aboveDesktop": "(min-width: 1440px)",
        "aboveLaptop": "(min-width: 1280px)",
        "abovePhone": "(min-width: 320px)",
        "abovePhoneMax": "(min-width: 414px)",
        "aboveTablet": "(min-width: 768px)",
        "aboveTabletMax": "(min-width: 960px)",
        "belowDesktop": "(max-width: 1440px)",
        "belowLaptop": "(max-width: 1280px)",
        "belowPhone": "(max-width: 320px)",
        "belowPhoneMax": "(max-width: 414px)",
        "belowTablet": "(max-width: 768px)",
        "belowTabletMax": "(max-width: 960px)",
        "landscape": "(orientation: landscape)",
        "portrait": "(orientation: portrait)",
      },
      "breakpointsVerbose": Object {
        "aboveDesktop": [Function],
        "aboveLaptop": [Function],
        "abovePhone": [Function],
        "abovePhoneMax": [Function],
        "aboveTablet": [Function],
        "aboveTabletMax": [Function],
        "belowDesktop": [Function],
        "belowLaptop": [Function],
        "belowPhone": [Function],
        "belowPhoneMax": [Function],
        "belowTablet": [Function],
        "belowTabletMax": [Function],
        "landscape": [Function],
        "portrait": [Function],
      },
      "colors": Object {
        "black": "#000000",
        "electricBlue": "#080CF1",
        "gray": Array [
          "#F3F3F3",
          "#F5F5F5",
          "#F8F8F8",
          "#E6E6E6",
          "#D5D5D5",
          "#CFCFCF",
        ],
        "gray0": "#F3F3F3",
        "gray1": "#F5F5F5",
        "gray2": "#F8F8F8",
        "gray3": "#E6E6E6",
        "gray4": "#D5D5D5",
        "gray5": "#CFCFCF",
        "lightBlue": Array [
          "#EEFCFF",
          "#BCE1F7",
          "#76C2F3",
        ],
        "lightBlue0": "#EEFCFF",
        "lightBlue1": "#BCE1F7",
        "lightBlue2": "#76C2F3",
        "lightPink": "#FBDDE4",
        "loading": "#BFBFBF",
        "navy": "#00003C",
        "pink": "#FEACBE",
        "red": "#FF511C",
        "rocketBlue": "#0073D1",
        "rocketBlueHover": "#0073EB",
        "shadyLady": "#979797",
        "white": "#FFFFFF",
        "yellow": "#F0FF02",
        "yellowHover": "#D8E320",
      },
      "fixedPosition": "relative",
      "fonts": Object {
        "headerFont": "\\"din-cond\\", \\"din-condensed-web\\", Arial, sans-serif",
        "primaryFont": "\\"ff-din-web\\", Helvetica, Arial, sans-serif",
        "secondaryFont": "\\"adobe-caslon-pro\\", Helvetica, Arial, serif",
      },
      "grid": Object {
        "columns": Object {
          "desktop": 12,
          "mobile": 4,
          "tablet": 12,
        },
        "gutter": 2,
        "margins": Object {
          "desktop": "7%",
          "mobile": "3%",
          "tablet": "7%",
        },
      },
      "media": Object {
        "desktop": [Function],
        "laptop": [Function],
        "phone": [Function],
        "phoneMax": [Function],
        "tablet": [Function],
        "tabletMax": [Function],
      },
    }
  }
/>
`;
